lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -212,7 +212,7 @@ class ClassMetadataInfo implements ClassMetadata
     /**
      * READ-ONLY: The name of the entity class.
      *
-     * @var string
+     * @var class-string
```

Comment:
Should we use `class-string<object>` wherever we use `class-string` but cannot be more precise?

Reply:
We could but this is implied. Psalm will interpret it exactly the same AFAIK: https://github.com/vimeo/psalm/blob/ab5ddb1514f7de0976b964e92e9bf51138aa1496/src/Psalm/Type/Atomic/TClassString.php#L31

Reply:
But you should use `@psalm-var` for class-string, because it's not a real php type.

Reply:
@orklah I know, my question is : should we say it anyway just like one might use the type declaration `mixed` in the future (https://wiki.php.net/rfc/mixed_type_v2)

Reply:
Well, this is a matter of preference. I guess it's kind of the same rationale as `mixed[]` (or `array<mixed>`) being prefered to `array`. I'll add it for now

Reply:
I never saw `class-string<object>`

Reply:
All things considered, we already have quite a few `class-string` in codebase and no `class-string<object>`. For coherence, I'll follow the de facto convention.